src/Compilers/VisualBasic/Portable/Syntax/SyntaxKindFacts.vb

```diff
@@ -22,6 +22,99 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 (SyntaxKind.AggregateKeyword <= kind AndAlso kind <= SyntaxKind.YieldKeyword)
         End Function
 
+        ''' <summary>
+        ''' Determine if the kind represents a control keyword
+        ''' </summary>
+        Public Shared Function IsControlKeyword(kind As SyntaxKind) As Boolean
+            Select Case kind
+                Case _
+                SyntaxKind.CaseKeyword,
+                SyntaxKind.CatchKeyword,
+                SyntaxKind.ContinueKeyword,
+                SyntaxKind.DoKeyword,
+                SyntaxKind.EachKeyword,
+                SyntaxKind.ElseKeyword,
+                SyntaxKind.ElseIfKeyword,
+                SyntaxKind.EndKeyword,
+                SyntaxKind.ExitKeyword,
+                SyntaxKind.FinallyKeyword,
+                SyntaxKind.ForKeyword,
+                SyntaxKind.GoToKeyword,
+                SyntaxKind.IfKeyword,
+                SyntaxKind.InKeyword,
+                SyntaxKind.LoopKeyword,
+                SyntaxKind.NextKeyword,
+                SyntaxKind.ResumeKeyword,
+                SyntaxKind.ReturnKeyword,
+                SyntaxKind.SelectKeyword,
+                SyntaxKind.ThenKeyword,
+                SyntaxKind.TryKeyword,
+                SyntaxKind.WhileKeyword,
+                SyntaxKind.WendKeyword,
```

Comment:
i didn't know 'wend' even existed!

Reply:
It will get flagged and suggest you replace with End While =)